tests/lit.site.cfg.in

```diff
@@ -5,6 +5,7 @@ config.name = 'LLVM regression suite'
 config.test_format = lit.formats.ShTest(False)
 config.suffixes = ['.ll', '.c', '.cpp', '.test', '.txt', '.s', '.mir']
 config.test_source_root = os.path.join("@CMAKE_CURRENT_SOURCE_DIR@", "tests")
+config.excludes = ['Inputs']
```

Comment:
I'm not a big fan of this `Inputs` directory. You could have suffixed your input file as `basic_lldb.in`, so that it's not considered a test.